tasks/base.py

```diff
@@ -1,24 +1,26 @@
 # -*- coding: utf-8 -*-
 """ tasks/base.py is for code used internally within the tasks package. """
+from utils.base import ImplementationError
+
 import logging
 import json
+from requests import Response
 
 from locust import TaskSet
 
-from utils.base import ImplementationError
```

Comment:
The `utils.base` import should still be at the very bottom, and `requests` should be grouped with `locust` (it's not a Python-base package, you have to install it)